packages/runtime/container-runtime/src/test/opLifecycle/OpGroupingManager.spec.ts

```diff
@@ -126,17 +126,14 @@ describe("OpGroupingManager", () => {
 				contentSizeInBytes: 0,
 				referenceSequenceNumber: 0,
 			};
-			assert.throws(
```

Comment:
Is this a use case for `validateAssertionError()`?

Reply:
seems like it! I just updated the test.